04_numeronym/mohitnag/main_test.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNumeronyms(t *testing.T) {
+	testData := []struct {
+		Scenario string
+		input    []string
+		expected []string
+	}{
+		{"Scenario One", []string{"ehi!@  45%!"}, []string{"e9!"}},
+		{"Scenario two", []string{""}, []string{""}},
+	}
+	for _, td := range testData {
+		td := td
+		t.Run(td.Scenario, func(t *testing.T) {
+			numeronym := numeronyms(td.input...)
+			assert.Equal(t, td.expected, numeronym)
+			fmt.Println(numeronym)
+		})
+	}
+}
+func TestMain(t *testing.T) {
+	assert := assert.New(t)
+	var buf bytes.Buffer
+	out = &buf
+	main()
+	expected := string("[a11y K8s abc]")
+	actual := buf.String()
```

Comment:
Since you have tested the logic in another Test function, main test can be made simpler to test a single one like `abc` or `abcd`

Reply:
Good point. However this is in line with the requirements : ```Call `fmt.Println(numeronyms("accessibility", "Kubernetes", "abc"))` in `main` to print:

```
[a11y K8s abc]
```
So i would like to keep it as it is .